health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -348,6 +390,222 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
+
+    // Extract sorted property names
+    const sortedPropertyNames = columns.map((column) => column.name);
+
+    // Update data with new properties and required fields
+    data.properties = properties;
+    data.required = required;
+
+  }
+
+  function convertIntoSchema(data) {
+    var convertData = { ...data };
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)